media-collection/public/media-non-automatic-session-handling.js

```diff
@@ -0,0 +1,299 @@
+const createSecondVideoPlayer = secondVideoPlayerId => {
```

Comment:
On this one you are getting the session id from the create media session call, and including that on all the events.